Bottom: 0d40ecd98b0349c5ea813c2414c2f9d7d3ffee17
Top:    a4d386fc22076f54793e3a7f911b42942d729481
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-15 16:52:04 -0700

Refresh of inspect-cmd

---
diff --git a/lldb/packages/Python/lldbsuite/test/commands/command/script/TestCommandScript.py b/lldb/packages/Python/lldbsuite/test/commands/command/script/TestCommandScript.py
index 88669374f77..852e54cbd60 100644
--- a/lldb/packages/Python/lldbsuite/test/commands/command/script/TestCommandScript.py
+++ b/lldb/packages/Python/lldbsuite/test/commands/command/script/TestCommandScript.py
@@ -16,12 +16,14 @@ class CmdPythonTestCase(TestBase):
     NO_DEBUG_INFO_TESTCASE = True
 
     def test(self):
-        #self.build()
+        self.build()
         self.pycmd_tests()
 
     def pycmd_tests(self):
         self.runCmd("command source py_import")
 
+        # test a bunch of different kinds of python callables with
+        # both 4 and 5 positional arguments
         self.expect("foobar", substrs=["All good"])
         self.expect("foobar4", substrs=["All good"])
         self.expect("vfoobar", substrs=["All good"])
@@ -33,8 +35,6 @@ class CmdPythonTestCase(TestBase):
         self.expect("cfoobar4", substrs=["All good"])
         self.expect("ifoobar4", substrs=["All good"])
 
-        return
-
         # Verify command that specifies eCommandRequiresTarget returns failure
         # without a target.
         self.expect('targetname',
